src/machine.rs

```diff
@@ -294,9 +293,6 @@ pub struct Evaluator<'mir, 'tcx> {
     /// The table of directory descriptors.
     pub(crate) dir_handler: shims::unix::DirHandler,
 
-    /// The "time anchor" for this machine's monotone clock (for `Instant` simulation).
-    pub(crate) time_anchor: Instant,
```

Comment:
Why did you move this to `thread`? We use to have monotone clock support before we had threads. That's a separate subsystem.

Reply:
because you need access to the global virtual clock inside `thread`. If we keep the `Arc` in `Time` then we need the `time_anchor` to clone it. If we remove the `Arc` then we need the `time_anchor` to call `get_wait_time`.